src/app/sagas/SettingsSaga.test.ts

```diff
@@ -50,16 +47,4 @@ describe("SettingsSaga", () => {
       });
     });
   });
-
-  describe("registerListeners", () => {
```

Comment:
Encounter error `Cannot set property fork of #<Object> which has only a getter` so I'm removing this test for now and will get back to it when time permits.